packages/core/src/Checkbox.js

```diff
@@ -57,7 +57,9 @@ class Checkbox extends PureComponent {
     }
 
     updateInputIndeterminate() {
-        this.inputRef.indeterminate = this.props.indeterminate;
+        if (this.inputRef) {
```

Comment:
This makes coverage drops slightly. Wanna cover this?

Reply:
OK, i will fix it

Reply:
This change is make for enzyme v3 previously. But i misunderstood the real reason that make the test failed, so I revert it back first.

In v3, `ref` callback will not fired in shallow test. Since `ref` is actual returned a DOM, but shallow test do not handled it. But shallow test still support some life cycle methods. (Check airbnb/enzyme#318)

So the better solution is use `mount()` API to test the component which use `ref` to some action.

Reply:
In next PR, I will switch the `shallow()` to `mount()` in checkbox test.